setup.py

```diff
@@ -51,13 +51,15 @@ def package_files(directory):
         "configsuite",
         "numpy",
         "pandas",
-        "six",
+        "six>=1.12.0",
         "scipy",
         "xlrd",
         "stea",
+        "pyscal>=0.4.0",
     ],
     setup_requires=["pytest-runner", "setuptools_scm"],
-    tests_require=["pytest", "mock", 'black; python_version>="3.6"'],
+    tests_require=["pytest", "mock", "openpyxl<=2.6.4", 'black; python_version>="3.6"'],
+    # The openpyxl restriction is relevant as long as Python2.7 is to be supported.
```

Comment:
I suggest we rephrase the `tests_require` as `'openpyxl<=2.6.4; python_version<"3"'` and remove the comment then ;) Better in code than in comments...

Reply:
fixed